scripts/test/run_selenium_tests_dev.sh

```diff
@@ -21,4 +21,5 @@ docker-compose run \
    -e ELASTICSEARCH_INDEX=testindex \
    -e USE_WEBPACK_DEV_SERVER=True \
    -e WEBPACK_DEV_SERVER_HOST="$WEBPACK_SELENIUM_DEV_SERVER_HOST" \
+   -e ELASTICSEARCH_DEFAULT_PAGE_SIZE=5 \
```

Comment:
You will need to do this in `run_selenium_tests_travis.sh` as well. You might be able to use `override_settings` too, it will be tricky due to the multiple threads